src/StepperFirmata2.cpp

```diff
@@ -87,7 +87,7 @@ boolean StepperFirmata2::handleSysex(byte command, byte argc, byte *argv)
   if (command == ACCELSTEPPER_DATA) {
 
     byte stepCommand, deviceNum, interface, wireCount, stepType;
-    byte stepOrMotorPin1, directionOrMotorPin2, motorPin3, motorPin4, enablePin, invertPins = 0;
+    byte stepOrMotorPin1, directionOrMotorPin2, ms1OrMotorPin3, ms2OrMotorPin4, enablePin, invertPins = 0;
```

Comment:
I'm not convinced that I want to include support for the MS1 and MS2 pins in this way. It's too specific to the EasyDriver. If you can find other stepper driver boards that use the same 2 pin convention, I could probably be convinced. Otherwise I'd prefer using standard DIGITAL_MESSAGE commands to set the 2 pin values from firmata.js. That could also be set in the Stepper2 constructor in firmata.js, just using DIGITAL_MESSAGE instead of Stepper2.config.